terraform/testflinger.tf

```diff
@@ -0,0 +1,84 @@
+terraform {
+  required_providers {
+    juju = {
+      version = "~> 0.7.0"
+      source  = "juju/juju"
+    }
+  }
```

Comment:
There'll be a need to describe a "backend" here also when you add state handling support in. I don't mind if that's in this PR or in a follow-up one.

Reply:
noted, still need to sort that bit out, but it can be part of a future update, or more likely as operational documentation with further details about staging/production deployments in our environment.